barretenberg/cpp/src/barretenberg/benchmark/plonk_bench/plonk.bench.cpp

```diff
@@ -32,21 +32,6 @@ plonk::Prover provers[NUM_CIRCUITS];
 plonk::Verifier verifiers[NUM_CIRCUITS];
 plonk::proof proofs[NUM_CIRCUITS];
 
-void construct_witnesses_bench(State& state) noexcept
```

Comment:
I've deleted a couple such "construct witness" benchmarks because they're not meaningful and `compute_witness` no longer exists. I imagine the original intent was to measure the time it takes to compute a circuit satisfying witness from scratch but this is just measuring the time it takes to create polynomials from pre-computed witness data.